test_app/tests/authentication/test_social_auth.py

```diff
@@ -47,6 +47,210 @@ def __unicode__(self):
     __repr__ = __unicode__
 
 
+@mock.patch("ansible_base.authentication.social_auth.logger")
+def test_authenticator_strategy_redirect_logging(mock_logger):
+    """Test that AuthenticatorStrategy.redirect logs the redirect URL without mocking redirect itself."""
+    from django.http import HttpResponseRedirect
+
+    strategy = AuthenticatorStrategy(storage=AuthenticatorStorage())
+    test_url = "https://example.com/oauth/callback"
+
+    # Call the redirect method directly (not mocked)
+    result = strategy.redirect(test_url)
+
+    # Verify the logger was called with the correct message
+    mock_logger.info.assert_called_once_with(f"Redirecting user to {test_url} as part of the social auth flow for SSO authenticator.")
+
+    # Verify that the result is an HttpResponseRedirect with the correct URL
+    assert isinstance(result, HttpResponseRedirect)
+    assert result.url == test_url
+
+
+@pytest.mark.django_db
```

Comment:
The `random_user` fixture is declared as a parameter but never used in this test. Consider removing it.

```suggestion
def test_social_auth_mixin_start_enabled_authenticator(mock_logger):
```